README.adoc

```diff
@@ -551,6 +551,31 @@ service.expects(:execute).returns(true)
 
 Choose only one to use – avoid mixing both approaches within one project.
 
+== Subclasses might run the parent tests methods twice.
+
+Minitest uses Ruby classes, if a Minitest class inherits from another class, it will also inherit its methods 
+causing Minitest to run the parent's tests twice.
+
+[source,ruby]
+----
+# example
+
+class ParentTest < Minitest::Test
+  def test_1
+    #... Run twice
+  end
+end
+
+class ChildTest < ParentTest
+  def test_2
+    #...
+  end
+end
+----
+
+In some cases, we want them to run the tests twice, but if not, avoid using subclasses or use minitest/spec, which disables 
+class inheritance between tests classes. You can measure how many duplicated tests you have using [this script](https://github.com/ignacio-chiazzo/ruby-minitest-analyzer).
```

Comment:
I think it's unlikely someone will switch to `minitest/spec` because of this. Also, I think it's probably not very common to _want_ to run the tests twice?

```suggestion
In rare cases, we may want to run the tests twice, but in general avoid subclassing tests cases.

Note: The `minitest/spec` alternative syntax disable inheritance between test classes and so does not have this behavior.
```

Reply:
Good point 👍